### PR TITLE
[DOC] Update section on new contributing and new CI flow process and other small edits

### PR DESCRIPTION
**Pull request by stdale-intel**
The CI system no longer auto runs pre-commit activities for new contributors by default. Update the documentation to reflect the new process of needing someone to approve workload run.  Also, since i'm here, make some small constancy edits in some files I was touching.

### CODE REVIEW

**Comment by stdale-intel** [comment]
@bader marking this as draft for now until we are ready to merge. Please feel free to provide any input on wording in the meantime.

File: CONTRIBUTING.md
Comment (bader) [inline CONTRIBUTING.md:83]: Can you confirm that this message is shown to the PR author? I know that it's visible to the maintainers, but I'm not sure how it looks like for the outside contributors.

**Comment by stdale-intel** [comment]
@aelovikov-intel any feedback from your side? else I am gonna proceed with merging this in.

**Review by aelovikov-intel — APPROVED** [review]

**Review by bader — APPROVED** [review]

**Review by bader — APPROVED** [review]

**Pull request merged**
